src/apify_client/clients/resource_clients/key_value_store.py

```diff
@@ -0,0 +1,152 @@
+import io
+import json
+from typing import Any, Dict, Optional
+
+from ..._errors import ApifyApiError
+from ..._utils import _catch_not_found_or_throw, _parse_date_fields, _pluck_data
+from ..base.resource_client import ResourceClient
+
+
+class KeyValueStoreClient(ResourceClient):
+    """Sub-client for manipulating a single key-value store."""
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Have you pls verified that when you are creating this client from other code that it gives you correct argument hints?

Reply:
Yup, it works for me in Visual Studio Code.